### DISCUSSION TITLE
how to bypass Sorry, this application cannot run under a Virtual Machine

### DISCUSSION THREAD

**Discussion by googleelgoog111** [utmapp/UTM#6149] [General]
I am trying to run a windows-based game on my 2022 arm macbook pro  on  UTM Windows11

However, I run into the issue of 'Sorry, this application cannot run under a virtual machine' 

 I have searched extensively for a solution, but does not work all.

I was wondering if anyone else had the same issue, and have managed to solve the issue?

Any help would be greatly appreciated.

**Comment by BokBoy** [comment]
Hmm probably impossible, unless you tweak and spoof the cpu id etc... 
I'm probably not allowed to say anything as it may result in legal issues but its a thought.

**Comment by that-ben** [comment]
Unless your game is an ARM binary running in ARM-based W11 (which I suspect is the case) it's not going to run well enough to even be playable.  But aside from that, why virtualize it in UTM when you could virtualize it straight in macOS using Apple's Game Porting Toolkit? I don't even know if your particular game is supported or not, but it may be worth a try? https://www.applegamingwiki.com/wiki/Game_Porting_Toolkit

**Comment by BokBoy** [comment]
> Unless your game is an ARM binary running in ARM-based W11 (which I suspect is the case) it's not going to run well enough to even be playable.  But aside from that, why virtualize it in UTM when you could virtualize it straight in macOS using Apple's Game Porting Toolkit? I don't even know if your particular game is supported or not, but it may be worth a try? https://www.applegamingwiki.com/wiki/Game_Porting_Toolkit

Apple's game porting toolkit isn't really worth it, really difficult for the general user.

**Comment by F1248** [comment]
This could help to play Windows games:
https://getwhisky.app

**Comment by that-ben** [comment]
Oh yeah! A mix between CrossOver and Apple's Game Porting Toolkit.  Basically, all three projects are related and use Wine at the core, then CodeWeavers' CrossOver behind the scene, then a nice UI on top.  Sadly, not all games are supported (yet?)

**Comment by BokBoy** [comment]
Nah ty whoever did that, I didn't know about whiskey (not alcohol)

**Comment by clauderoy790** [comment]
Did you find a solution? I'm having the exact problem with same OS trying to run Knight Online on a Win 11 VM from VMWare Fusion
